File: Assets/_Project/Scripts/Utils/API Services/Models/Banner.cs
using System.Collections.Generic;

[System.Serializable]
public class BannerItem
{
    public string _id { get; set; }
    public string url { get; set; }
    public string image { get; set; }
    public bool isActive { get; set; }
    public string bannerType { get; set; }
    public int? bannerSequence { get; set; }

}


[System.Serializable]
public class Banners : BaseModel<List<BannerItem>>
{

}

File: RummyChampion_V1-main/src/models/Banner.js
const mongoose = require('mongoose');

const bannerSchema = new mongoose.Schema({
    url: {
        type: String,
        required: true
    },
    image: {
        type: String, // URL or path to the image file
        required: true
    },
    createdOn: {
        type: Date,
        default: Date.now
    },
    isActive: {
        type: Boolean,
        default: true
    },
    bannerType: {
        type: String
    },
    bannerSequence: {
        type: Number
    }
}, {
    timestamps: true,
    versionKey: false
});

const Banner = mongoose.model('Banner', bannerSchema);

module.exports = Banner;
